composer.json

```diff
@@ -3,7 +3,7 @@
     "description": "secucard connect PHP client SDK",
     "license": "Apache-2.0",
     "homepage": "https://github.com/secucard/secucard-connect-php-sdk",
-    "version": "1.14.1",
+    "version": "1.14.2",
```

Comment:
culd you please use the version 1.15.0. The last digit is in general for bug-fixes.